octokit_utils.rb

```diff
@@ -37,15 +38,15 @@ class OctokitUtils
     'puppet_authorization',
     'reboot',
     'registry',
-    'satellite',
+    'satellite_pe_tools',
```

Comment:
@tphoney Looking at the regex on line 51, looks like it requires the full name so 👍